### ISSUE TITLE
Provide shortcut for configuring DEFAULT_FIREBASE provider

### ISSUE THREAD

**Issue by jeffbcross** [angular/angularfire2#24]
```js
providers: [defaultFirebase('https://myfirebase')]
``` 

instead of 

```js
providers: [provide(DEFAULT_FIREBASE, {useValue: 'https://myfirebase'})]
```

**Comment by jeffbcross** [comment]
FYI I renamed `DEFAULT_FIREBASE` to `FirebaseUrl`, and `DEFAULT_FIREBASE_REF` to `FirebaseRef`. So the function name `defaultFirebase` could be improved.

**Comment by jeffbcross** [comment]
Closed by dddd0b3646e5afe7f9e068f6158e3c62c6c3552c
